opacclient/opacapp/src/main/java/de/geeksfactory/opacclient/frontend/MultiStepResultHelper.java

```diff
@@ -124,7 +124,7 @@ public void askForConfirmation(final MultiStepResult result) {
             ((ViewGroup) view.findViewById(R.id.rlConfirm))
                     .removeView(table);
             TextView tv = new TextView(context);
-            tv.setText(result.getDetails().get(0)[0]);
+            tv.setText(Html.fromHtml(result.getDetails().get(0)[0].replaceAll("uuml","uuml;")));
```

Comment:
This special case should be handled within the Adis API implementation, the text in `result.getDetails().get(0)[0]` doesn't need to contain any HTML.

Reply:
Also, to be more specific, we should probably replace `"&uuml" with "&uuml;", instead of `"uuml"` with `"uuml;"``